### ISSUE TITLE
Rpm repository configuration

### ISSUE THREAD

**Issue by olenagerasimova** [artipie/artipie#227]
Add settings section for rpm repository configuration: 
```yaml
settings:
  digest: sha256
  naming-policy: sha1
  filelists: true
```
Digest parameter - optional, default sha256, configures `Digest` instance for Rpm: sha256 or sha1.
Naming-policy parameter - optional, default plain, configures `NamingPolicy` for Rpm: plain, sha256 or sha1.
Filelists parameter - optional, default true, includes File Lists for Rpm: true or false.

**Comment by olenagerasimova** [comment]
@0crat assign me

**Comment by 0crat** [comment]
> @0crat assign me ([here](https://github.com/artipie/artipie/issues/227#issuecomment-646526062))

@olenagerasimova The job [#227](https://github.com/artipie/artipie/issues/227) assigned to @olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova), here is [why](https://www.0crat.com/footprint/CT2E6TK9B/87336700-8ce6-4f2e-826d-14981dca3b1e); the budget is 30 minutes, see [§4](http://www.zerocracy.com/policy.html#4); please, read [§8](http://www.zerocracy.com/policy.html#8) and [§9](http://www.zerocracy.com/policy.html#9); if the task is not clear, read [this](http://www.yegor256.com/2015/02/16/it-is-not-a-school.html) and [this](http://www.yegor256.com/2015/01/15/how-to-cut-corners.html); there will be no monetary reward for this job

**Comment by 0crat** [comment]
@olenagerasimova[/z](https://www.0crat.com/u/olenagerasimova) this job was assigned to you 5days ago. It will be taken away from you soon, unless you close it, see [§8](http://www.zerocracy.com/policy.html#8). Read [this](http://www.yegor256.com/2014/04/13/no-obligations-principle.html) and [this](http://www.yegor256.com/2014/11/24/principles-of-bug-tracking.html), please.

**Comment by olenagerasimova** [comment]
@0crat wait please for #242 t be merged

**Comment by olenagerasimova** [comment]
closed via #242
